src/pages/MyKiva/MyKivaPage.vue

```diff
@@ -240,7 +240,12 @@ export default {
 			});
 
 			if (this.goalsEntrypointEnable) {
-				const { showRenewedAnnualGoalToast } = await this.renewAnnualGoal();
+				// Just for demo purposes, force goals renew in my kiva
+				// TODO: Remove forced renewYear after testing MP-2316
+				const { renewYear } = this.$route.query;
+				const { showRenewedAnnualGoalToast } = await this.renewAnnualGoal(
+					renewYear ? new Date(`${renewYear}-01-15T00:00:00Z`) : undefined
```

Comment:
The thought is that we'd be overriding the current year with `?renewYear=2026` in the URL?